### DISCUSSION TITLE
Equivalent of `advancedFIDOConfig` in `verifyAuthenticationResponse` for `verifyRegistrationResponse`?

### DISCUSSION THREAD

**Discussion by agektmr** [MasterKale/SimpleWebAuthn#599] [Ideas]
I wonder why `verifyAuthenticationResponse` has `advancedFIDOConfig` while `verifyRegistrationResponse` does not. It would be handy if they are symmetric.

**Comment by MasterKale** [comment]
Hey @agektmr, good question! `advancedFIDOConfig` only came to be because of FIDO Conformance testing. There's a FIDO-specific requirement that an RP support **optional User Presence** during authentication and so I added that in to be able to pass those tests:

> [Several](https://github.com/MasterKale/SimpleWebAuthn/issues/246) [issues](https://github.com/MasterKale/SimpleWebAuthn/issues/248) [have](https://github.com/MasterKale/SimpleWebAuthn/issues/252) come in recently about failing FIDO Conformance tests. While ideologically I disagree with the tests that require support for non-`true` authentication (they technically violate [the WebAuthn spec which requires `up` to always be true](https://www.w3.org/TR/webauthn-2/#sctn-op-get-assertion)), users of my library may wish to implement FIDO conformant RP servers with it. 

From https://github.com/MasterKale/SimpleWebAuthn/issues/253#issue-1338580903

I continue to disagree with the importance of supporting this use case but alas, FIDO Conformance is what it is 🤷‍♂️

**Reply by MasterKale** [reply]
Interestingly there's no such requirement for registration so `advancedFIDOConfig` never ended up in `verifyRegistrationResponse()` 🤔

**Reply by agektmr** [reply]
Thanks for clarification, Matt. I understood. I thought it might be handy to add `advancedFIDOConfig` to `verifyRegistrationResponse()` but you are right, supporting "uv=preferred" on the server doesn't make sense.
